data-prepper-plugins/geoip-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/extension/AwsAuthenticationOptionsConfig.java

```diff
@@ -1,9 +1,9 @@
 /*
  * Copyright OpenSearch Contributors
- * SPDX-License-Identifier: Apache-2.0
+ *  PDX-License-Identifier: Apache-2.0
```

Comment:
This should be `SPDX`. See: https://github.com/opensearch-project/data-prepper/blob/main/CONTRIBUTING.md#java-gradlegroovy

Maybe this was a replace-all error?

Reply:
This change was applied to a few files.

Reply:
It's just a typo in my Intellij copyright setup, fixed it.